### DISCUSSION TITLE
Customizable Bounce Message

### DISCUSSION THREAD

**Discussion by tiredofit** [stalwartlabs/smtp-server#33] [Q&A]
Q: Are there plans for a customizable bounce message? 

I tend to use customized (and very slightly different) bounce messages per MTA to differentiate easily for support purposes in postfix by means of `bounce_template_file`.

**Comment by mdecimus** [comment]
Not currently but feel free to add an enhancement request on Github. You can file an enhancement request for any modern feature that Postfix (or other MTA) supports and Stalwart doesn't.

**Reply by tiredofit** [reply]
Thanks! I'll file an enhancement request and see if others see useful as well.
